guides/common/modules/proc_upgrading-a-connected-project-server.adoc

```diff
@@ -158,6 +158,7 @@ endif::[]
 ----
 # dnf -y module switch-to postgresql:13
 ----
+If you are using an external database, upgrade to PostgreSQL 13.
```

Comment:
This should be an own step, and it should be rendered for Satellite too (which it currently is not because for Satellite we use foreman-maintain which takes care of all those switches)